docs/components/concepts/encryption-at-rest.md

```diff
@@ -0,0 +1,74 @@
+| Software encryption key           | Camunda    | <p><ul><li><p>Google KMS [software](https://cloud.google.com/docs/security/key-management-deep-dive#software_backend_software_protection_level) protection level.</p></li><li>Cryptographic operations are performed in software.</li><li>Compliant with [FIPS 140-2 Level 1](https://cloud.google.com/docs/security/key-management-deep-dive#fips_140-2_validation).</li></ul></p>                |
+| Hardware encryption key           | Camunda    | <p><ul><li><p>Google KMS [hardware](https://cloud.google.com/docs/security/key-management-deep-dive#backend_hardware_protection_level) protection level.</p></li><li>Cryptographic operations are performed in a hardware security module (HSM).</li><li>Compliant with [FIPS 140-2 Level 3](https://cloud.google.com/docs/security/key-management-deep-dive#fips_140-2_validation).</li></ul></p> |
+
+## Provider encryption key (default)
+
+By default, Camunda 8 SaaS cluster data at rest is protected using Google Cloud Platform (GCP) encryption.
+
+- Provider encryption keys are owned and managed by GCP.
+- Google uses a [FIPS 140-2](https://cloud.google.com/security/compliance/fips-140-2-validated) validated encryption module.
+
+:::info
+Learn more about [Google default encryption at rest](https://cloud.google.com/docs/security/encryption/default-encryption) and default provider encryption settings.
+:::
+
+## Camunda-managed software encryption key
+
+Camunda-managed software encryption keys use the Google KMS [software](https://cloud.google.com/docs/security/key-management-deep-dive#software_backend_software_protection_level) protection level to provide a higher level of protection than default provider encryption.
+
+- Requires an enterprise plan.
+- Software encryption keys are managed by Camunda.
+- Software encryption keys are compliant with [FIPS 140-2 Level 1](https://cloud.google.com/docs/security/key-management-deep-dive#fips_140-2_validation).
+- Cryptographic operations are performed in software.
+
+## Camunda-managed hardware encryption key
+
+Camunda-managed hardware encryption keys use the Google KMS [hardware](https://cloud.google.com/docs/security/key-management-deep-dive#backend_hardware_protection_level) protection level to provide a higher level of protection than both default provider encryption and Camunda-managed software encryption keys.
+
+- Requires an enterprise plan.
+- Hardware encryption keys are managed by Camunda.
+- Hardware encryption keys are compliant with [FIPS 140-2 Level 3](https://cloud.google.com/docs/security/key-management-deep-dive#fips_140-2_validation).
+
+## Key rotation
+
+Camunda-managed software and hardware encryption keys are rotated annually with zero downtime to maintain a high level of security and compliance.
```

Comment:
I guess at least anually since it could happen that we need to rotate due to a security incident or similar?